### PR TITLE
Images search endpoint toggle

### PR DESCRIPTION
**Pull request by jamieparkinson**
This is the - obviously very minimal - first pass at getting the new images endpoint used on .org. Because images and works are different types it was a bit fiddly, I duplicated some files so as to be as flexible as possible and allow for further development.

Resolves https://github.com/wellcomecollection/wellcomecollection.org/issues/5295

### CODE REVIEW

File: catalogue/webapp/components/ExpandedEndpointImage/ExpandedEndpointImage.js [hunk 1]
```diff
@@ -0,0 +1,272 @@
+// @flow
```
Comment (jamieparkinson) [inline catalogue/webapp/components/ExpandedEndpointImage/ExpandedEndpointImage.js:1]: This file is copied from `ExpandedImage`

File: catalogue/webapp/components/ImageEndpointSearchResults/ImageEndpointSearchResults.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
+// @flow
```
Comment (jamieparkinson) [inline catalogue/webapp/components/ImageEndpointSearchResults/ImageEndpointSearchResults.js:1]: This file is copied from `ImageSearchResults`

File: catalogue/webapp/services/catalogue/images.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+type GetImagesProps = {|
+  params: CatalogueImagesApiProps,
+  pageSize?: number,
+  ...Environment,
+|};
+
+export async function getImages({
+  params,
+  env = 'stage', // TODO switch this to prod once the images search is deployed
```
Comment (jamieparkinson) [inline catalogue/webapp/services/catalogue/images.js:19]: We are having some issues with the pipeline that is delaying deploying the images search stuff to the prod API. So as not to delay this, and as it's behind a toggle, I thought I'd put this here for now.

**Comment by jamieparkinson** [comment]
Something else that will need to be thought about is what to display in the panel now we don't have a title, description, or a link to the image within the work. We might want to add some or all of these to the images model depending on what we decide cc @jennpb @jtweed 

image

**Comment by jtweed** [comment]
The intent was always to display the data from the source work, showing the context for where the image comes from, then letting you jump off to that.

**Comment by jamieparkinson** [comment]
> I wonder if we could simplify some of the conditional work going on by the works page by having a SearchResultsPage that is then extended by WorksSearchResults and ImagesSearchResults...

I just spent quite a while trying to get this to work and concluded that doing it in such a way that would satisfy Flow is a bit too tricky... However I did realise that I'd made a couple of mistakes!

**Pull request merged**